db/schema.rb

```diff
@@ -10,14 +10,14 @@
 #
 # It's strongly recommended that you check this file into your version control system.
 
-ActiveRecord::Schema[7.2].define(version: 2024_11_26_104331) do
+ActiveRecord::Schema[8.0].define(version: 2024_11_29_151352) do
   # These are extensions that must be enabled in order to support this database
   enable_extension "btree_gin"
   enable_extension "btree_gist"
   enable_extension "citext"
   enable_extension "pg_buffercache"
+  enable_extension "pg_catalog.plpgsql"
```

Comment:
I'm not sure why this is here. Something from Rails 8? @gms-gs

Reply:
Actually nevermind. 🤔 It is a different extension